Apps/WebClient/src/ClientApp/app/views/landing.vue

```diff
@@ -73,6 +73,55 @@
       margin-left: auto;
       margin-right: auto;
       margin-top: -40px;
+
+      .icons {
+        display: flex;
+        flex: 0 0 auto;
+        flex-direction: column;
+      }
+
+      .iconNames {
+        flex-direction: column;
+        display: flex;
+        flex: 1 1 auto;
+      }
+
+      .status-active {
+        display: inline-block;
+        color: $primary;
+        line-height: 40px;
+        height: 40px;
+        vertical-align: middle;
+        .icon {
+          text-align: center;
+          background-color: $primary;
+          color: white;
+          border-radius: 50%;
+          width: 40px;
+        }
+      }
+
+      .status-inactive {
+        color: darkgray;
+        display: inline-block;
+        line-height: 40px;
+        height: 40px;
+        vertical-align: middle;
+        .icon {

```

Comment:
this one and the previous icon classes are pretty much the same. Lets move them outside the active/inactive wrappers